vscode/src/test/unit/index.ts

```diff
@@ -0,0 +1,52 @@
+import * as Mocha from 'mocha';
```

Comment:
we can probably follow same convention like runTest.ts which we did for integration